main.go

```diff
@@ -159,26 +122,25 @@ func Main() {
 
 	// As websocket client already checks if the cloud token is set, we can start it here.
 	go RunWebsocketClient()
-	initPublicIPState()
 
+	initPublicIPState()
 	initSerialPort()
 
 	setProcTitle("ready")
+	mainLogger.Log().Msg("JetKVM ready")
+
+	go RunAutoUpdateCheck()
 
 	sigs := make(chan os.Signal, 1)
 	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
 	<-sigs
 
-	logger.Log().Msg("JetKVM Shutting Down")
-	//if fuseServer != nil {
-	//	err := setMassStorageImage(" ")
-	//	if err != nil {
-	//		logger.Infof("Failed to unmount mass storage image: %v", err)
-	//	}
-	//	err = fuseServer.Unmount()
-	//	if err != nil {
-	//		logger.Infof("Failed to unmount fuse: %v", err)
-	//	}
+	if err := rpcUnmountImage(); err != nil {
```

Comment:
Have to unmount the drive or the UsbGadget gets wonky... and it takes _reboot_ to get it back.